javascript/index.js

```diff
@@ -35,31 +45,71 @@ function printSplit() {
 }
 
 function clearSplits() {
+ 
   // ... your code goes here
 }
 
 function setStopBtn() {
+
+  btnLeft.innerHTML ='STOP'
+  btnLeft.setAttribute = ('class', 'btn stop')// wont change color, but split does. why?
```

Comment:
when working around classes for DOM elements it is better to use classList or className.

` btnLeft.className = "btn stop"`

or ...

`btnLeft.classList.toggle("stop")`
`btnLeft.classList.toggle("start")`